_includes/foot-idr.html

```diff
@@ -17,13 +17,19 @@
                     <p>OMERO is distributed under the terms of the GNU GPL. For more information, visit <a href="https://www.openmicroscopy.org">openmicroscopy.org</a></p>
                 </div>
                 <hr class="whitespace">
-                <div class="small-12 columns text-center">
-                    <p class="version-number"><a href="{{ site.baseurl }}/index.html"><img id="version-number" src="{{ "/img/logos/logo-idr.svg" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="IDR logo"></a>
+                <div>
+                    <div class="version-number float-left">
+                        <a href="{{ site.baseurl }}/index.html"><img id="version-number" src="{{ "/img/logos/logo-idr.svg" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}" alt="IDR logo"></a>
```

Comment:
We might want to update this URL to point at the home page rather than the about page like the header but this is not a blocker for this PR (and the logo is quite buried at the bottomof the page anyways)